libvuln/opts.go

```diff
@@ -154,6 +160,18 @@ func (o *Opts) parse(ctx context.Context) error {
 	return nil
 }
 
+func remoteMatchersFunc(o *Opts) error {
+	for _, m := range o.RemoteMatchers {
+		if m["name"] == "crda" {
+			m, err := crda.NewMatchers(m["params"])
+			if err == nil {
+				o.Matchers = append(o.Matchers, m)
+			}
+		}
+	}
+	return nil
+}
```

Comment:
crda string shouldn't be hard coded. It should behave like a updaterSet.
Updaterset registers Updater construction function with name & calls it based on the value passed through opts.